<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>822</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="StatusTab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>Status</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3" rowminimumheight="0,0,0">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item row="2" column="0">
         <widget class="QGroupBox" name="commonDataBusAndRegisterResultStatusGroupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Common Data Bus and Register Result Status</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="topMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <property name="bottomMargin">
            <number>10</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="sizeConstraint">
              <enum>QLayout::SetDefaultConstraint</enum>
             </property>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::MinimumExpanding</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="clockLabelStatusTab">
               <property name="text">
                <string>Clock Count:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="clockCountLabelStatusTab">
               <property name="text">
                <string>0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::MinimumExpanding</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="commonDataBusAndRegisterStatusTableWidget">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <attribute name="horizontalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="reservationStatusGroupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Reservation Status</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="topMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <property name="bottomMargin">
            <number>10</number>
           </property>
           <item>
            <widget class="QTableWidget" name="functionalUnitReservationStatusTableWidget">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>2</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <attribute name="horizontalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="memoryReservationStatusTableWidget">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <attribute name="horizontalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QGroupBox" name="instrucitonStatusGroupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Instruction Status</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="topMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <property name="bottomMargin">
            <number>10</number>
           </property>
           <item row="0" column="0">
            <widget class="QTableWidget" name="instructionStatusTableWidget">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="lineWidth">
              <number>1</number>
             </property>
             <attribute name="horizontalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ExecutionTableTab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>Execution Table</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QTableWidget" name="executionTableWidget">
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>593</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuApplication">
    <property name="title">
     <string>Application</string>
    </property>
    <widget class="QMenu" name="menuFiles">
     <property name="title">
      <string>Files</string>
     </property>
     <addaction name="actionLoad_Files"/>
     <addaction name="actionUnload_Files"/>
    </widget>
    <addaction name="menuFiles"/>
    <addaction name="separator"/>
    <addaction name="actionClose_Application"/>
   </widget>
   <widget class="QMenu" name="menuArchitecture">
    <property name="title">
     <string>Architecture</string>
    </property>
    <addaction name="actionCreate_Architecture"/>
    <addaction name="actionEdit_Architecture"/>
    <addaction name="actionLoad_Architecture"/>
   </widget>
   <widget class="QMenu" name="menuInstruction_List">
    <property name="title">
     <string>Instruction List</string>
    </property>
    <addaction name="actionCreate_Instruction_List"/>
    <addaction name="actionEdit_Instruction_List"/>
    <addaction name="actionLoad_Instruction_List"/>
   </widget>
   <widget class="QMenu" name="menuScript">
    <property name="title">
     <string>Script</string>
    </property>
    <addaction name="actionCreate_Script"/>
    <addaction name="actionEdit_Script"/>
    <addaction name="actionLoad_Script"/>
   </widget>
   <widget class="QMenu" name="menuAlgorithm_2">
    <property name="title">
     <string>Algorithm</string>
    </property>
    <widget class="QMenu" name="menuStart_2">
     <property name="title">
      <string>Start</string>
     </property>
     <addaction name="actionManual_Step_Algorithm"/>
     <addaction name="actionClock_Step_Algorithm"/>
     <addaction name="actionFull_Speed_Algorithm"/>
    </widget>
    <addaction name="actionPrerun_Check"/>
    <addaction name="menuStart_2"/>
    <addaction name="actionStep_Algorithm"/>
    <addaction name="actionPause_Algorithm"/>
    <addaction name="actionReset_Algorithm"/>
   </widget>
   <addaction name="menuApplication"/>
   <addaction name="menuArchitecture"/>
   <addaction name="menuInstruction_List"/>
   <addaction name="menuScript"/>
   <addaction name="menuAlgorithm_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClose_Application">
   <property name="text">
    <string>Close Application</string>
   </property>
  </action>
  <action name="actionCreate_Architecture">
   <property name="text">
    <string>Create</string>
   </property>
  </action>
  <action name="actionLoad_Architecture">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionEdit_Architecture">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionCreate_Instruction_List">
   <property name="text">
    <string>Create</string>
   </property>
  </action>
  <action name="actionEdit_Instruction_List">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionLoad_Instruction_List">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionCreate_Script">
   <property name="text">
    <string>Create</string>
   </property>
  </action>
  <action name="actionEdit_Script">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionLoad_Script">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionLoad_Files">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionUnload_Files">
   <property name="text">
    <string>Unload</string>
   </property>
  </action>
  <action name="actionPrerun_Check">
   <property name="text">
    <string>Prerun Check</string>
   </property>
  </action>
  <action name="actionStep_Algorithm">
   <property name="text">
    <string>Step</string>
   </property>
  </action>
  <action name="actionPause_Algorithm">
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionReset_Algorithm">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionManual_Step_Algorithm">
   <property name="text">
    <string>Manual Step</string>
   </property>
  </action>
  <action name="actionClock_Step_Algorithm">
   <property name="text">
    <string>Clock Step</string>
   </property>
  </action>
  <action name="actionFull_Speed_Algorithm">
   <property name="text">
    <string>Full Speed</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
